documentation/quarkus/getting-started-quarkus.asciidoc

```diff
@@ -0,0 +1,233 @@
+
+
+== Testing a Quarkus Application 
+After we have built the service, we have to verify it with some tests.
+We will give you some guidelines to implement some test cases.
+
+.Testing Guides 
+. General testing guide + 
+For users that aren't familiar with the devon4j testing principles, we created a general link:../guide-testing.asciidoc[best practices and recommendations guide for testing].
+.. Our guide for testing with Quarkus
+In addition, we also provide a guide that specifically addresses the link:guide-for-testing.asciidoc[testing of a Quarkus application].
+  
+Most of the Quarkus applications are already equipped with a basic Test and also our https://github.com/devonfw-sample/devon4quarkus-reference[reference project] provides some test cases, if you want to improve and extends the tests, you can also follow the large https://quarkus.io/guides/getting-started-testing[Quarkus guide for testing]. 
+
+
+== How to Integrate a Database
+The next step for our REST service would be to integrate a database to store the objects of the entity. 
+
+With Quarkus, adding a database can be easy, because Quarkus can take over the build-up and connection process.
+First, you should understand our guides to the concepts of how to work with data and then we will show how to integrate a database with Quarkus.
+
+.Data Principles Guides
+. General devon4j JPA guide + 
+To get an insight into the general JPA usage you should read the link:../guide-jpa.asciidoc[JPA guide] which contains a general explanation of the Java Persistence API.
+. Difference to SpringData + 
+If you have already worked with SpringData this is also partially supported with Quarkus, this is explained in more detail in this link:../guide-repository.asciidoc[SpringData Guide].
+
+
+.Database Integration 
+. Quarkus zero config dev mode + 
+Starting with database implementation in Quarkus, we recommend for beginners to use the https://quarkus.io/guides/datasource#dev-services[DEV mode Zero Config Setup (Dev Services)] this is especially great for testing the code without a database set up. 
+Quarkus does all the work for you and configures a database and creates the database and tables(schemas) for you.
+.. Configuration Properties + 
+A list of all database configuration properties for the https://quarkus.io/guides/datasource#configuring-dev-services[Dev services]
+. Integrate a simple Hibernate ORM database + 
+The zero config setup only works with the Dev mode, it's comfortable in the first phases of the creation of your service but if the goal is to also get a deployable version, you have to create your own database and integrate it.
+This Quarkus guide shows, how to integrate a https://quarkus.io/guides/hibernate-orm[Hibernate ORM database] with an example service.
+.. Configuration list for JDBC + 
+A list of all configuration that is possible with a https://quarkus.io/guides/datasource#jdbc-configuration[JDBC configuration properties]
+. Hibernate ORM with Panache + 
```

Comment:
I am not familiar with the features and usage of Panache. But I think since we are focusing on Spring Data JPA in devon4j, this should be choice to go. If we provide too many solutions, it will be a bit misleading for the reader

Reply:
If I understand it right, it's an implementation of a Hibernate ORM that we recommend also with SpringData? I will remove it, but it seems that Panache is also a powerful tool with Quarkus.